src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/routing/web/WebReportRouteConfig.kt

```diff
@@ -52,6 +52,11 @@ object WebReportRouteConfig : RouteConfig
                     .json()
                     .secure(runReports)
                     .transform(),
+            WebEndpoint("/running/:key/logs/",
```

Comment:
We already have an endpoint for:
```
/report/:name/actions/status/:key/
```
so maybe this one should be:
```
/report/:name/actions/logs/:key/
```
I know we don't need the name, but neither does the status endpoint.